jobserver/management/commands/osi_release.py

```diff
@@ -52,31 +53,6 @@ def create_release(*, backend, path, user):
     return r.json()["release_id"]
```

Comment:
`url = f"{BASE_URL}workspace/opensafely-internal-interactive"`

Is this being hard-coded going to cause any problems, given that I'm using a different workspace?

Reply:
Yes, yes it is, fixed 👍